apis/Google.Datastore.V1Beta3/Google.Datastore.V1Beta3.Snippets/DatastoreDbSnippets.cs

```diff
@@ -90,10 +91,11 @@ public void StructuredQuery()
             {
                 Filter = Filter.Equal("author", "Jane Austen")
             };
-            DatastoreQueryResults results = db.RunQuery(query);
-            // DatastoreQueryResults implements IEnumerable<Entity>, but you can
-            // call AsEntityResults(), AsBatches() or AsResponses() to see the query
-            // results in whatever way makes most sense for your application.
+            LazyDatastoreQuery results = db.RunQuery(query);
```

Comment:
As an aside, returning an `IEnumerator<T>` is pretty useless in .NET:

- I can't iterate over it with `foreach`
- I need to remember to dispose of it
- I can't use LINQ with it, or anything else that will use `IEnumerable<T>`

It definitely appeals in terms of expressing the "iterate once", but it's just not idiomatic :(